src/gluonts/model/deepar/_estimator.py

```diff
@@ -127,9 +128,12 @@ def __init__(
             num_eval_samples > 0
         ), "The value of `num_eval_samples` should be > 0"
         assert dropout_rate >= 0, "The value of `dropout_rate` should be >= 0"
-        assert all(
-            [c > 0 for c in cardinality]
-        ), "Elements of `cardinality` should be > 0"
+        assert (
+            cardinality is not None or not use_feat_static_cat
+        ), "You must set `cardinality` if `use_feat_static_cat=True`"
```

Comment:
Could we just use categorical features when cardinality is set and drop the field otherwise?
Currently `use_feat_static_cat` is redundant.

Reply:
we could, but this would be a bit asymmetric